demo/examples/panoviewer/projection-type/cubemap_image.html

```diff
@@ -64,7 +64,10 @@ <h2>Support variety cubemap format</h2>
 
     var panoViewer = new PanoViewer(container, {
         projectionType: "cubemap",
-        image: "../../img/test_cube_3x2_RLUDFB.jpg"
+        image: "../../img/test_cube_3x2_RLUDFB.jpg",
+        cubemapConfig: {
+          gap: 3
```

Comment:
Is the gap a value that the user should properly set?

Reply:
I think we can't find value automatically as it's texture issue.